src/components/schedule-page/table/schedule/schedule-parts/schedule-row.component.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useCallback, useContext, useEffect, useMemo, useState } from "react";
```

Comment:
Tego pliku nie musi być. Zmieniliśme jego nazwę na base-row.component, bo taka jest o wiele bardziej sensowna :D 
Sprawdzałem i u ciebie na branchu też jest ten plik, więc jedyne co potrzeba zrobić- to usunąc `schedule-row.component`
![image](https://user-images.githubusercontent.com/21079319/99724280-af178980-2ab3-11eb-88f2-83a10b37be28.png)